src/metabase/lib/metadata/jvm.clj

```diff
@@ -351,27 +351,16 @@
                :active          true
                :visibility_type [:not-in #{"sensitive" "retired"}])
 
-
     :metadata/metric
-    (t2/select :metadata/metric :table_id table-id, :type :metric, :archived false)
+    (t2/select :metadata/metric :table_id table-id, :source_card_id [:= nil], :type :metric, :archived false)
```

Comment:
Can `table_id` and `source_card_id` be set at the same time? I was thinking they should be mutually exclusive.
If they can't should there be a db constraint?
If they can, this could be easy to forget.